prac_08/unreliable_car_test.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 | Prac_08 | Dannielle Jones
+Unreliable car test client program.
+"""
+from prac_08.unreliable_car import UnreliableCar
+
+# Should return being driven.
```

Comment:
Since it's random, there's only a chance it will be one or the other - not guaranteed.
Multiple tests would help here.